node-rdkafka 2.3.0 Segmentation fault

Since 2.3.0, as soon as `connect()` method is called (from consumer or producer), a Segmentation fault occured.

Steps to reproduce using docker and e2e tests for brevity : 

**Dockerfile :**
```Dockerfile
FROM node:9-stretch
RUN npm i node-rdkafka
WORKDIR /node_modules/node-rdkafka
RUN npm i
ENV KAFKA_HOST=localhost:29092
```
**e2e tests run :**
```bash
$ docker build -t rdkafka .
(...)

$ docker run --net host rdkafka make e2e
  Consumer
    commit
Makefile:63: recipe for target 'e2e' failed
make: *** [e2e] Segmentation fault (core dumped)
```

--- a/binding.gyp
+++ b/binding.gyp
@@ -64,6 +64,7 @@
                       'OS=="linux"',
                       {
                         "libraries": [
+                          "<(module_root_dir)/build/deps/librdkafka.so",
                           "<(module_root_dir)/build/deps/librdkafka++.so",
                           "-Wl,-rpath=<(module_root_dir)/build/deps",
                         ],

